lte/gateway/python/setup.py

```diff
@@ -86,10 +86,10 @@
         'scripts/spgw_service_cli.py',
         'scripts/cpe_monitoring_cli.py',
         'scripts/state_cli.py',
-        'scripts/dhcp_helper_cli.py',
         'scripts/dp_probe_cli.py',
         'scripts/user_trace_cli.py',
         'scripts/icmpv6.py',
+        'dhcp_helper_cli/dhcp_helper_cli.py',
```

Comment:
iiuc this will add the script to the magma debian package.

Reply:
That would be an issue. Maybe we don't need it with the symlink in the magma-dev VM.